client/src/pages/Login.js

```diff
@@ -176,28 +202,28 @@ const Login = ({ isLoginForm }) => {
           icon={<FacebookIcon />}
           onClick={() => handleSocialLogin("facebook")}
         >
-          Facebook
+          <ButtonText>Facebook</ButtonText>
         </SocialButton>
         <SocialButton
```

Comment:
- The corners are cut off because `border-radius: 0.8rem`, just need to make the border radius be unset or zero 
- On Figma it says the border width is 1px, but for reason currently it looks really thin even though it is 1px... can we try increasing the border width a bit?

Current:
<img width="195" alt="Screen Shot 2020-05-03 at 9 32 20 PM" src="https://user-images.githubusercontent.com/42228386/80936451-1bd06e00-8d86-11ea-99a5-dffd1722b64c.png">

Figma:
<img width="492" alt="Screen Shot 2020-05-03 at 9 37 07 PM" src="https://user-images.githubusercontent.com/42228386/80936489-3efb1d80-8d86-11ea-945f-f6c6a7d88322.png">

Reply:
Nothing changes when I increase the border width... I can't figure out how to target it.

Reply:
Yea sometimes it's hard overriding ANTD styles, might just need to be more specific with the css selector in order to target it. ALso, need to change the `border-radius` not `border`


<img width="1061" alt="Screen Shot 2020-05-03 at 10 32 37 PM" src="https://user-images.githubusercontent.com/42228386/80938445-ffd0ca80-8d8d-11ea-867f-25e54dde3720.png">

<img width="213" alt="Screen Shot 2020-05-03 at 10 31 16 PM" src="https://user-images.githubusercontent.com/42228386/80938399-d57f0d00-8d8d-11ea-96e6-fa1a77b82d55.png">

Reply:
But if it's a bit difficult we can leave it (as per my comment previous to this)

Reply:
I tried again but no luck... I think creating new reusable buttons is the way to go